src/components/[guild]/activity/ActivityLogContext.tsx

```diff
@@ -152,6 +193,9 @@ const ActivityLogProvider = ({
     data: transformActivityLogInfiniteResponse(ogSWRInfiniteResponse.data),
     mutate: () => ogSWRInfiniteResponse.mutate(),
     isUserActivityLog: !!userId,
+    withActionGroups: withActionGroups,
```

Comment:
Since the keys and values have the same name, we can use the short version here:

```ts
const value = {
    ...ogSWRInfiniteResponse,
    data: transformActivityLogInfiniteResponse(ogSWRInfiniteResponse.data),
    mutate: () => ogSWRInfiniteResponse.mutate(),
    isUserActivityLog: !!userId,
    withActionGroups,
    actionGroup,
    setActionGroup,
  }
```